Cargo.toml

```diff
@@ -1,3 +1,5 @@
+cargo-features = ["strip"]
```

Comment:
Wow. I didn't know this. Thanks!

Reply:
It's a nightly feature in Cargo: https://github.com/rust-lang/cargo/pull/8246